### ISSUE TITLE
Use without express?

### ISSUE THREAD

**Issue by brodybits** [mkschreder/node-php#13]
I need a quick CLI to run WordPress for testing and had already hacked https://github.com/brodybits/node-php-serve which uses node-php-server. I found this project through https://github.com/alangecker/express-php, which seems to be a more compact rewrite in CoffeeScript. (It references your project as the source of the "env" headers fixing code.) It would be nice if I could use this project without express.

A couple of other issues:

For some reason, I had to modify my copy of main.js to use `/usr/local/bin/php-cgi` instead of `php-cgi`. I wonder if the `php-cgi` path (absolute path and/or search path) should be an option (with a default value)?

Another minor issue is that when I run this project with WordPress, I get the following warnings:
```shell
express deprecated res.send(body, status): Use res.status(status).send(body) instead node_modules/node-php/main.js:115:13
express deprecated res.sendfile: Use res.sendFile instead node_modules/node-php/main.js:120:12
```

I did *not* get these errors with express-php.

I suspect this will need some modifications-please correct me in case I am mistaken. I can think of the following possibilities, assuming that I am right, in increasing order of preference:
- I keep my own fork without the express part
- Make the express part optional in this project
- Drop the express support from this project and either give some shim code or point to express-php for those who need the express support

**Comment by mkschreder** [comment]
You should use community versions of this package. People have modified and
improved it since I first published it 2 years ago. I don't think I'm the
right person to keep it updated anymore because I don't have time to debug
it. So you can modify it in whichever way you want and release your own
spinoffs with or without express. If you seriously want to maintain it
moving forward, and do a good job at giving people something that works, I
can add a note in my repo and tell everyone that you're currently
maintaining latest version of it.

**Comment by brodybits** [comment]
Thanks for the response. I am having *much* better luck with https://github.com/alangecker/express-php, with some fixes in alangecker/express-php#3 to get it working with WordPress.

Unfortunately I still have not received a response from @alangecker to my query in alangecker/express-php#1 (comparison with this project) or alangecker/express-php#2 (small fix that I needed for Homebrew). I am afraid that I will have to maintain my own fork.

I do not think it is worth the effort to get rid of the `express` dependency at this stage. I found https://github.com/pillarjs/router which seems to be a simpler version of `express` but it will need some more adjustments to `express-php`.
